ci_build_images/sles.Dockerfile

```diff
@@ -17,7 +17,9 @@ RUN zypper -n update \
     && ARCH=$(rpm --query --queryformat='%{ARCH}' zypper) \
     && if [ "$ARCH" = x86_64 ]; then ARCH=amd64 ; fi \
     && zypper addrepo https://ci.mariadb.org/galera/mariadb-4.x-latest-gal-"${ARCH}-${ID%%leap}-${VERSION_ID}".repo \
-    && zypper -n install \
+    # permissions -> https://bugzilla.opensuse.org/show_bug.cgi?id=1228968 workaround \
+    && zypper -n install permissions; \
```

Comment:
is this step necessary?

Reply:
its allowing the permission package that was previously pulled in by a dependency to fail its post install step, but still be counted as installed. While the above bug mentioned a package update, it wasn't sufficient to get around this, so hopeing its open status means there's still more work coming. There's still an opensuse pip failure in the CI below, so we may have a flaky bit for a bit.

Reply:
oh, did not read the comment! OK then